sound/soc/codecs/rt700.c

```diff
@@ -256,6 +255,10 @@ static void rt700_jack_detect_handler(struct work_struct *work)
 			SND_JACK_HEADSET |
 			SND_JACK_BTN_0 | SND_JACK_BTN_1 |
 			SND_JACK_BTN_2 | SND_JACK_BTN_3);
+	snd_soc_jack_report(rt700->hs_jack, rt700->jack_type,
+			SND_JACK_HEADSET |
+			SND_JACK_BTN_0 | SND_JACK_BTN_1 |
+			SND_JACK_BTN_2 | SND_JACK_BTN_3);
```

Comment:
for my education, why do we call snd_soc_jack_report() twice? I don't get what the code does here, maybe a comment would help?